univocity-trader-core/src/test/java/com/univocity/trader/indicators/RAVIIndicatorTest.java

```diff
@@ -0,0 +1,62 @@
+package com.univocity.trader.indicators;
+
+import com.univocity.trader.candles.CandleHelper;
+import org.junit.Test;
+
+import static com.univocity.trader.indicators.base.TimeInterval.minutes;
+import static org.junit.Assert.assertEquals;
+
+public class RAVIIndicatorTest {
+
+    @Test
+    public void ravi() {
+        RAVIIndicator indicator = new RAVIIndicator(3, 8, minutes(1));
+
+        indicator.process(CandleHelper.newCandle(0, 0, 110, 0, 0), 0, true);
```

Comment:
don't call the method `process` in unit tests. This is not a public method. Use `indicator.accumulate(candle)` instead.

Also for each new candle, increment the timestamp (parameter i, the first parameter in `CandleHelper.newCandle`, you are using 0 on everything.)